newIDE/app/src/InAppTutorial/InAppTutorialOrchestrator.js

```diff
@@ -746,21 +797,39 @@ const InAppTutorialOrchestrator = React.forwardRef<
         const {
           objectName,
           sceneName: layoutName,
+          count,
         } = objectSceneInstancesToWatch;
         const layout =
           layoutName && project.hasLayoutNamed(layoutName)
             ? project.getLayout(layoutName)
             : project.getLayoutAt(0);
-        const instances = layout.getInitialInstances();
-        if (instances.hasInstancesOfObject(objectName)) {
-          goToNextStep();
+        if (!count) {
+          // If no count is provided, we just check if there is at least one instance.
+          const instances = layout.getInitialInstances();
+          if (instances.hasInstancesOfObject(objectName)) {
+            goToNextStep();
+          }
+        } else {
+          // Otherwise, we check if there is the expected number of instances.
+          const instancesCount = getInstancesInLayoutForObject(
```

Comment:
You can use `getInstancesCountInLayoutForObject`

Reply:
that's useful!